sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -2419,6 +2435,26 @@ private <T extends Resource> Flux<FeedResponse<T>> createReadManyQuery(
         return executionContext.flatMap(IDocumentQueryExecutionContext<T>::executeAsync);
     }
 
+    private <T> Flux<FeedResponse<Document>> createPointReadOperations(
+        Map<PartitionKeyRange, CosmosItemIdentity> singleItemPartitionRequestMap,
+        String resourceLink,
+        CosmosQueryRequestOptions queryRequestOptions,
+        Class<T> klass
+    ) {
+        return Flux.fromIterable(singleItemPartitionRequestMap.values())
+            .flatMap(item -> {
+                CosmosQueryRequestOptions clonedQueryRequestOptions = ModelBridgeInternal.createQueryRequestOptions(queryRequestOptions);
+                clonedQueryRequestOptions.setPartitionKey(item.getPartitionKey());
+                return this.readDocument(getDocumentLink(item.getId(), resourceLink), ModelBridgeInternal.toRequestOptions(clonedQueryRequestOptions));
```

Comment:
`getDocumentLink()` is only used once here - better to just inline it with `resourceLink + item.getId()` and remove the `getDocumentLink()` method.